### DISCUSSION TITLE
Solargraph [Help Wanted]

### DISCUSSION THREAD

**Discussion by khandudo** [NvChad/NvChad#2410] [Q&A] [answered]
While using solargraph as lsp (for ruby):

1. "LSP implementation" -> <gi>
2. "LSP references" -> <gr>

Doesn't seems to work at all. The cursor just sit there on the text and doesn't respond with anything. Goto-Defination seems to work though.

after <gr>

<img width="1074" alt="Screenshot 2023-09-30 at 11 54 06 PM" src="https://github.com/NvChad/NvChad/assets/70809089/c0b9b69b-e217-4f05-a223-f492e9450a47">

Any kinda would be highly appreciated. 🙏

**Comment by siduck** [comment, marked as answer]
not a nvchad issue tho, nvchad doesnt even manage those things. Its lspconfig & neovim which does it, ask it at r/neovim!
